installing/installing_bare_metal_ipi/ipi-install-installation.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ipi-install-installation"]
+= Installing OpenShift
+include::_attributes/common-attributes.adoc[]
+:context: ipi-install-installation
+
+toc::[]
+
+// Deploying the cluster via the {product-title} installer
+include::modules/ipi-install-deploying-the-cluster-via-the-openshift-installer.adoc[leveloffset=+1]
+
+// Following the installation
+include::modules/ipi-install-following-the-installation.adoc[leveloffset=+1]
+
+// Verifying static IP address configuration
+include::modules/ipi-install-verifying-static-ip-address-configuration.adoc[leveloffset=+1]
+
+// Preparing to reinstall a cluster on bare metal
+include::modules/ipi-preparing-reinstall-cluster-bare-metal.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_creating_manifest_ignition"]
+== Additional resources
+* xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installation-user-infra-generate-k8s-manifest-ignition_installing-bare-metal[{product-title} Creating the Kubernetes manifest and Ignition config files]
```

Comment:
```suggestion
* xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installation-user-infra-generate-k8s-manifest-ignition_installing-bare-metal[Creating the Kubernetes manifest and Ignition config files]
```
I don't think you need to mention OCP here since the customers will already be in the OCP docs. If you do choose to keep it, be sure to change "Creating" to lowercase.